tests/test_models.py

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python
+"""
+Tests for the `event-bus-kafka` models module.
+"""
```

Comment:
One of the failing CI checks is:

```
ERROR: InvocationError for command /home/timmc/edx-repos/event-bus-kafka/.tox/py38-django32/bin/pytest (exited with code 5)
```

Code 5 is "no tests were run", which is a failure because it usually means the test config is broken. One option would be to add a placeholder here:

```
def test_placeholder():
    """Placeholder test to allow pytest to succeed before real tests are in place."""
    pass
```

Reply:
(And I should probably put that into the cookiecutter as well...)

Reply:
(For reference, a proposed change to cookiecutter: https://github.com/openedx/edx-cookiecutters/compare/timmc/tests?expand=1#diff-a2b18a277d418a38b34c73e3de3ff486fcf7c5821cfc584831781220b44930d1R20)